src/vm/gcenv.ee.cpp

```diff
@@ -1082,17 +1081,58 @@ bool GCToEEInterface::GetIntConfigValue(const char* key, int64_t* value)
       GC_NOTRIGGER;
     } CONTRACTL_END;
 
+    if (strcmp(key, "GCSegmentSize") == 0)
```

Comment:
Do we really need to special case these 3 config settings for CoreCLR here?

Reply:
the idea was we didn't want to do the rest at all if it's one of the ones the VM already did the work to parse and has the info for.